cborattr/include/cborattr/cborattr.h

```diff
@@ -85,6 +86,9 @@ struct cbor_array_t {
         struct {
             double *store;
         } reals;
+        struct{
+            int *store;
+        } halffloats;
```

Comment:
I never used a half-float before, but I'd assume it is 16-bit long (half a float!), so why use `int` instead of `uint16_t`?

Reply:
The thinking here was to use standard C types, looking further on in the code I can see uint16_t being used. I've updated the halffloat to be a uint16_t type.